.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -62,16 +66,15 @@ jobs:
           MAGMA_DEV_MEMORY_MB: 9216
         working-directory: lte/gateway
         run: |
-          fab --show=debug --set DOCKER_REGISTRY=${DOCKER_REGISTRY} integ_test_containerized:test_mode="selected_tests",tests="TESTS=s1aptests/test_attach_detach.py"
-
+          fab --show=debug --set DOCKER_REGISTRY=${DOCKER_REGISTRY} integ_test_containerized:test_mode=${{ matrix.test_targets }}
       - name: Get test results
         if: always()
+        working-directory: lte/gateway
         run: |
-          cd lte/gateway
-          fab get_test_summaries:dst_path="test-results,sudo_tests=False"
+          fab get_test_summaries:dst_path="test-results",sudo_tests=False,dev_vm_name="magma_deb"
       - name: Upload test results
```

Comment:
Could we also publish the test results after this step?

Reply:
Yes, but lets do it in a separate follow up PR, ok?